lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1236,6 +1264,16 @@ instance LiftHandler ErrSubmitExternalTx where
             , errReasonPhrase = errReasonPhrase err400
             }
 
+instance LiftHandler ErrForgetPendingTx where
+    handler = \case
+        ErrForgetPendingTxNoSuchWallet e -> handler e
+        ErrForgetPendingTxNoSuchTransaction e -> handler e
+        ErrForgetPendingTxTransactionIsNotPending tid ->
+            apiError err404 TransactionNotPending $ mconcat
+                [ "The transaction with id : ", pretty tid,
```

Comment:
`pretty` will show a truncated transaction id, in the API response, we probably want to show the full id so I'd advise for `toText` instead.

Reply:
good point - fixed